### PR TITLE
add kueue e2e tests for kubernetes 1.27 

### PR DESCRIPTION
**Pull request by trasc**
Add kueue e2e tests for kubernetes 1.27.

### CODE REVIEW

File: config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml [hunk 1]
```diff
@@ -58,7 +58,7 @@ presubmits:
           limits:
             cpu: "4"
             memory: "16Gi"
-  - name: pull-kueue-test-e2e-main-1-24
```
Comment (alculquicondor) [inline config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml:61]: I think we can keep it for this release. 1.24 will be in support when we release kueue 0.4.

Comment (tenzen-y) [reply config/jobs/kubernetes-sigs/kueue/kueue-presubmits-main.yaml:61]: +1

**Review by alculquicondor — COMMENTED** [review]
/approve

**Comment by alculquicondor** [comment]
/hold cancel
@tenzen-y leaving lgtm to you

**Review by tenzen-y — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**